3.5.3-beta/Dockerfile

```diff
@@ -22,17 +22,19 @@ RUN set -ex; \
     mkdir -p "$ZOO_DATA_LOG_DIR" "$ZOO_DATA_DIR" "$ZOO_CONF_DIR"; \
     chown "$ZOO_USER:$ZOO_USER" "$ZOO_DATA_LOG_DIR" "$ZOO_DATA_DIR" "$ZOO_CONF_DIR"
 
+ARG GPG_KEY=C61B346552DC5E0CB53AA84F59147497767E7473
 ARG DISTRO_NAME=zookeeper-3.5.3-beta
 
 # Download Apache Zookeeper, verify its PGP signature, untar and clean up
 RUN set -ex; \
     apk add --no-cache --virtual .build-deps \
-        gnupg; \
-    wget -q "http://www.apache.org/dist/zookeeper/$DISTRO_NAME/$DISTRO_NAME.tar.gz"; \
-    wget -q "http://www.apache.org/dist/zookeeper/$DISTRO_NAME/$DISTRO_NAME.tar.gz.asc"; \
-    wget -q "http://www.apache.org/dist/zookeeper/KEYS"; \
+        gnupg openssl ca-certificates && update-ca-certificates; \
```

Comment:
Please make it multiline and sorted like that https://github.com/docker-library/logstash/blob/master/5/alpine/Dockerfile#L36-L38

Is this `update-ca-certificates` really required? What does it do? I haven't seen this in a single official image.

Reply:
It was also suggested to download from an SSL protected site, so we need to install openssl and update the certs on the container.

Reply:
Yeah, I understand why we need `ca-certificates`. But there is no need to call `update-ca-certificates`, because it's done automatically by `apk` ([proof](https://git.alpinelinux.org/cgit/aports/tree/main/ca-certificates/ca-certificates.trigger)).